src/Network/Network/Models/AzureFirewall/PSAzureFirewall.cs

```diff
@@ -70,6 +72,12 @@ public string NetworkRuleCollectionsText
             get { return JsonConvert.SerializeObject(NetworkRuleCollections, Formatting.Indented); }
         }
 
+        [JsonIgnore]
+        public string ThreatIntelWhitelistText
```

Comment:
What is this for?

Reply:
This is for display in Powershell for the ThreatIntelWhitelist property. Just as NetworkRuleCollections and ApplicationRuleCollections